lynx@2.9.0dev.6: download via aria2 failed

I changed the username because i dont think it would make any difference, and for privacy reasons

```
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'cacert' (2021-05-25) [64bit]
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 38dbd4|OK  |   1.4MiB/s|C:/Users/MY_USERNAME/scoop/cache/cacert#2021-05-25#https_curl.haxx.se_ca_cacert-2021-05-25.p
em_cacert.pem
Download: Status Legend:
Download: (OK):download completed.
Checking hash of cacert-2021-05-25.pem ... ok.
Linking ~\scoop\apps\cacert\current => ~\scoop\apps\cacert\2021-05-25
Running post-install script...
'cacert' (2021-05-25) was installed successfully!
Installing 'lynx' (2.9.0dev.6) [64bit]
Starting download with aria2 ...
Download: 06/06 20:27:37 [ERROR] CUID#9 - Download aborted. URI=https://slproweb.com/download/Win32OpenSSL_Light-1_1_1j.
Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://slproweb.com/download/Win32OpenSSL_Light-1_1_1j.ex
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 66fcbc|ERR |       0B/s|C:/Users/MY_USERNAME/scoop/cache/lynx#2.9.0dev.6#https_slproweb.com_download_Win32OpenSSL_Li
ght-1_1_1j.exe
Download: ca9e1f|OK  |   1.0MiB/s|C:/Users/MY_USERNAME/scoop/cache/lynx#2.9.0dev.6#https_invisible-island.net_datafiles_curren
t_lynx-newssl-setup.exe
Download: Status Legend:
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.
ERROR Download failed! (Error 3) Resource was not found
ERROR https://invisible-island.net/datafiles/current/lynx-newssl-setup.exe
    referer=https://invisible-island.net/datafiles/current/
    dir=C:\Users\MY_USERNAME\scoop\cache
    out=lynx#2.9.0dev.6#https_invisible-island.net_datafiles_current_lynx-newssl-setup.exe
https://slproweb.com/download/Win32OpenSSL_Light-1_1_1j.exe
    referer=https://slproweb.com/download/
    dir=C:\Users\MY_USERNAME\scoop\cache
    out=lynx#2.9.0dev.6#https_slproweb.com_download_Win32OpenSSL_Light-1_1_1j.exe

ERROR & 'C:\Users\MY_USERNAME\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\MY_USERNAME\scoop\cache\lynx.txt' --user-a
gent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto
-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn
 --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.
2 --stop-with-process=10744 --continue --summary-interval 0

```

--- a/bucket/lynx.json
+++ b/bucket/lynx.json
@@ -10,11 +10,11 @@
     "depends": "cacert",
     "url": [
         "https://invisible-island.net/datafiles/current/lynx-newssl-setup.exe",
-        "https://slproweb.com/download/Win32OpenSSL_Light-1_1_1j.exe"
+        "https://slproweb.com/download/Win32OpenSSL_Light-1_1_1L.exe"
     ],
     "hash": [
-        "a800c2d808bf2d90738c98da3f8df55f824f6934d775addc5ca9fcb4c56e2e39",
-        "sha512:76647e6c45a1f201c3e3125d5961f63fd84997b0d24f65da8797c6bf740f7e7745a746dfee8281bfb3342892864f4dd8bb099dfcb09397af93ded268c318269c"
+        "434d81bc288f529a2129ea83c3deb0389bad5b9430864ac475274365daf6eca5",
+        "sha512:75b31fcffeffd25caadbf5e0a9a71c001d1a38ce50bae8c3077b140cedce023c35f05fdfb2e29d9656ed26daa71348018efe0911df72a9bd0e1c641293eea0cb"
     ],
     "innosetup": true,
     "pre_install": "if (!(Test-Path \"$persist_dir\\lynx.cfg\")) { Add-Content \"$dir\\lynx.cfg\" -Value @(\"SSL_CERT_FILE:`\"$(appdir cacert $global)\\current\\cacert.pem`\"\", \"FORCE_SSL_PROMPT:PROMPT\") -Encoding Ascii }",

